examples/javascript/cloudflare-worker/README.md

```diff
@@ -0,0 +1,36 @@
+# Logfire Cloudflare Worker Integration Example
+
+This example is based on the scaffolding created from `npm create cloudflare@latest`, and uses the [otel-cf-workers package](https://github.com/evanderkoogh/otel-cf-workers) to instrument a Cloudflare Worker and send traces and metrics to Logfire.
+
+## Run the project
+
+To run the example locally, you need a Logfire account and a project and a local installation of Node 20. If you don't have a Logfire account, create a free one [in Logfire](https://logfire.pydantic.dev/).
+
+Clone the repository:
+
+```bash
+git clone https://github.com/pydantic/logfire.git
+```
+
+`cd` into the example directory and install:
+
+```bash
+cd examples/javascript/cloudflare-worker
+npm install
+```
+
+Edit the `wrangler.toml` file and add your Logfire project ID and API key:
+
+```toml
+[vars]
+LOGFIRE_WRITE_TOKEN="your-write-token"
+LOGFIRE_BASE_URL="https://logfire.pydantic.dev/"
```

Comment:
It needs to be `https://logfire-api.pydantic.dev/`. And it should just be hard coded, or changed to an OTEL env var.